src/__tests__/third-party/@apollo/client/__fixtures__/lib/apolloClient.ts

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
This fixture was copied from https://github.com/vercel/next.js/tree/canary/examples/with-apollo. Why? Because its quite complex and it should cover some real world scenarios.